cmd/fluxsvc/main.go

```diff
@@ -234,7 +235,7 @@ func main() {
 		logger.Log("addr", *listenAddr)
 		mux := http.NewServeMux()
 		mux.Handle("/metrics", promhttp.Handler())
-		handler := transport.NewHandler(server, transport.NewRouter(), logger)
```

Comment:
It would be nice to use the common http server at some point for common logging and instrumentation, but fine for now.